jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -70,23 +33,42 @@ describe('EntityDetails', function() {
 
   it('can be rendered', function() {
```

Comment:
I tried that. When doing a shallow render, getEntity is never invoked so the output is always the "Loading..." message. When doing a deep render, the child `<EntityHeader>` is also rendered so the output is complex and deep again. Since this refactor is blocking other work, my inclination is to land as is and then discuss how to improve the test in a separate PR.